upgrade-notes/backend_20240818_204612.md

```diff
@@ -0,0 +1,4 @@
+#### {limited user can't user free transport and gateway payments ([#3355](https://github.com/shopsys/shopsys/pull/3355))
+
+-   method `Shopsys\FrameworkBundle\Model\Payment\PaymentPriceCalculation::getCalculatedPricesIndexedByPaymentId` was removed
```

Comment:
It feels like there should be more BC changes mentioned - e.g. `IndependentPaymentVisibilityCalculation` or `PaymentPriceCalculation` constructor changes, please double check that

Reply:
fixed: https://github.com/shopsys/shopsys/pull/3355/commits/05d2ac3aa432acc837d8e7235b89423dc0b18600